src/Mailjet/Message/AdherentLeaveMembershipMessage.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\Adherent;
+use Ramsey\Uuid\Uuid;
+
+final class AdherentLeaveMembershipMessage extends MailjetMessage
```

Comment:
Cancel doesn't seem truly right. `TerminateMembership` seems to be the best according to Google translate.

"terminate membership" is the translation for "résilier l'adhésion"